File: OOAD/OOADEntity/OOADEntity/ManageFunds.cpp
#include "ManageFunds.h"

bool ManageFunds::AddFunds(float amt, Gambler g)
{
	if (g.AddFunds(amt))
		return true;
	return false;
}

bool ManageFunds::RemoveFunds(float amt, Gambler g)
{
	if (g.WithdrawFunds(amt))
		return true;
	return false;
}

File: OOAD/CasinoManagement/CasinoManagement/ManageFunds.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CasinoManagement
{
    class ManageFunds
    {
        public void AddFunds (float amt, Gambler g)
        {
            g.AddFunds(amt);
        }

        public void RemoveFunds(float amt, Gambler g)
        {
            g.RemoveFunds(amt);
        }
    }
}
